app/Jobs/CreateDuesPaymentDueNotificationInNova.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable SlevomatCodingStandard.ControlStructures.EarlyExit.EarlyExitNotUsed
+
+namespace App\Jobs;
+
+use App\Models\User;
+use App\Notifications\Nova\DuesPaymentDue;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Laravel\Nova\Notifications\Notification;
+
+class CreateDuesPaymentDueNotificationInNova implements ShouldQueue, ShouldBeUnique
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    private User $user;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(User $user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        if ($this->user->dues()->pending()->count() > 0 &&
+            $this->user->hasPermissionTo('access-nova') &&
+            0 === Notification::where('notifiable_type', '=', User::class)
+                ->where('notifiable_id', '=', $this->user->id)
+                ->where('type', DuesPaymentDue::class)
+                ->where('created_at', '>', now()->subMonths(3))
+                ->count()
+        ) {
```

Comment:
Might be a decent candidate for a query scope on the User class?

Reply:
I don't think a query scope is quite the right refactor, but I will make it an Eloquent relationship.